content/community.html

```diff
@@ -27,6 +27,14 @@
     <p>
       <strong>Issue tracker:</strong> We use the GitHub issue tracker for the various <a href="http://github.com/prometheus">Prometheus repositories</a>
     </p>
+    <p>
+      <em>Please refrain from soliciting individual project members
+      for support. Use the channels above instead, where you will find
+      more potential helpers, and the whole community can benefit from
```

Comment:
"where the whole community can help you and benefit from the solutions provided"